lib/media/track/mediatrack.js

```diff
@@ -218,17 +221,20 @@ class MediaTrack extends Track {
     const mediaStream = el.srcObject;
     if (mediaStream instanceof this._MediaStream) {
       mediaStream.removeTrack(this.mediaStreamTrack);
-      // NOTE(mroberts): It's as if, in Chrome and Safari, the <audio> element's
-      // `srcObject` setter is taking a "snapshot" of the MediaStream's
-      // MediaStreamTracks in order to playback; hence, calls to `removeTrack`
-      // don't take effect unless you set the <audio> element's `srcObject` again.
-      //
-      //   https://bugs.chromium.org/p/chromium/issues/detail?id=749928
-      //
-      el.srcObject = mediaStream;
     }
 
     this._attachments.delete(el);
+
+    if (this._attachments.size === 0) {
+      // NOTE(mpatwardhan): chrome continues playing audio track
+      // even after audio element is removed from the DOM.
+      // Removing track from the stream does not work either.
+      // https://bugs.chromium.org/p/chromium/issues/detail?id=749928
+      // So here we mark the track.enabled = false when there are
+      // no elements attached to it.
+      this.mediaStreamTrack.enabled = false;
```

Comment:
@makarandp0 What if we just set `el.srcObject = null` here instead, and set it back to a MediaStream when `attach()` is called? Wouldn't that work as well?

Reply:
@manjeshbhargav, I experimented with this. It would typically work, but not in our case because we have  '_dummyEl'  which also would need to be fixed to point its srcObject to `null`

It seems like  `enabled` property fits our purpose really well here.

Reply:
@makarandp0 I'm worried about disabling the MediaStreamTrack because it might affect the emission of the "started" event on the RemoteTrack. Also, this does not cover the case when the RemoteTrack is created and no attachments are made yet. Plus this code is shared with LocalTrack as well.

Reply:
Thanks, very good points - I will check about the `started` property to make sure its not affected by setting 'enable' to false. Also I will look into moving this for remote track only.

Reply:
Thanks @manjeshbhargav for raising these concerns - I have now verified that 

1) mediatrack do get started notification - I forced the track `enabled=false`, before receiving the 'started' event, and verified that we still get `started` event.  I think that's because when marked as disabled, mediaStreamTrack still pumps frames into mediaStream but just empty ones. 

2)  I also verified that the track does not play when there are no attachments are made. I guess the problem is mainly about chrome not handling dom removals of audio elements, and maintaining some kind of global state for webrtc tracks. 

3) Third part - about this code handling LocalTrack - thats good point but it's  matter of refactoring the code. If we agree to this solution then I can go tackle 3. 

I agree that although this solution is simple and elegant, it may have side effects, but this property is really promising and documentation does indicate this to be correct usage of the property - I am hoping that we can discover any issues in our testing.